runhouse/cli_utils.py

```diff
@@ -0,0 +1,523 @@
+
+    # print headline
+    daemon_headline_txt = (
+        "\N{smiling face with horns} Runhouse Daemon is running \N{Runner}"
+    )
+    console.print(daemon_headline_txt, style="bold royal_blue1")
+
+    console.print(f"[reset]Runhouse v{status_data.get('runhouse_version')}")
+    print_cloud_properties(cluster_config)
+    console.print(f"[reset]server pid: {status_data.get('server_pid')}")
+
+    # Print relevant info from cluster config.
+    print_cluster_config(cluster_config)
+
+    # print general cpu and gpu utilization
+    cluster_gpu_utilization: float = status_data.get("server_gpu_utilization")
+
+    # cluster_gpu_utilization can be none, if the cluster was not using its GPU at the moment cluster.status() was invoked.
+    if cluster_gpu_utilization is None and has_cuda:
+        cluster_gpu_utilization: float = 0.0
+
+    cluster_cpu_utilization: float = status_data.get("server_cpu_utilization")
+
+    server_util_info = (
+        f"CPU Utilization: {round(cluster_cpu_utilization, 2)}% | GPU Utilization: {round(cluster_gpu_utilization, 2)}%"
+        if has_cuda
+        else f"CPU Utilization: {round(cluster_cpu_utilization, 2)}%"
+    )
+    console.print(server_util_info)
+
+    # print the environments in the cluster, and the resources associated with each environment.
+    print_envs_info(env_servlet_processes, current_cluster)
+
+
+def print_bring_cluster_up_msg(
+    cluster_name: str, msg_prefix="Can't execute the command"
+):
+    from runhouse.main import console
+
+    console.print(
+        f"{msg_prefix} because [reset]{cluster_name} is not up. To bring it up, run [bold italic]`runhouse cluster up {cluster_name}`[/bold italic]."
+    )
+
+
+def get_cluster_or_local(cluster_name: str):
+    from runhouse.main import console
+
+    if cluster_name:
+        current_cluster = rh.cluster(name=cluster_name)
+        if not current_cluster.is_up():
+            console.print(
+                f"Cluster [reset]{cluster_name} is not up. If it's an on-demand cluster, you can run "
+                f"[reset][bold italic]`runhouse cluster up {cluster_name}`[/bold italic] to bring it up automatically."
+            )
+            raise typer.Exit(1)
+        try:
+            if current_cluster._http_client:
+                current_cluster._http_client.check_server()
+        except requests.exceptions.ConnectionError:
+            console.print(
+                f"Could not connect to the server on cluster {cluster_name}. Check that the server is up with "
+                f"[reset][bold italic]`runhouse cluster status {cluster_name}`[/bold italic] or [bold italic]`sky status -r`[/bold italic] for on-demand clusters."
+            )
+            raise typer.Exit(1)
+        return current_cluster
+
+    cluster_or_local = rh.here
+    if cluster_or_local == "file" and not cluster_name:
+        # If running outside the cluster must specify a cluster name
+        console.print("Missing argument `cluster_name`.")
```

Comment:
forget if this was already addressed somewhere else, but just to be safe this can `raise typer.Exit(1)` instead of returning, and then we can remove all these: 

```
if not current_cluster:
   return
```

Reply:
It was updated in the server cli commands revamp PR, updating here as well.